qna/DE-bring-your-own-tse.md

```diff
@@ -0,0 +1,31 @@
+## Question
+
+Wie sieht der Prozess aus wenn ich bereits eine neue TSE besitze und diese mit fiskaltrust verwenden möchte? 
+
+## Question
+
+Kann ich mit einer gebrauchten TSE zu fiskaltrust wechseln?
+
+## Question
+
+Wie kann ich meine (neue/gebrauchte) TSE im Portal registrieren?
+
+## Question
+
+Wie kann ich meine (neue/gebrauchte) TSE im Portal konfigurieren?
+
+## Question
+
+Ändert sich für mich als Händler etwas am Verkaufsprozess von Produkten von fiskaltrust? 
+
+## Metadata tags
+
+lang-de, market-de, PosOperator PosDealer, Consultant
+
+## Answer
+
+Der Prozess ist immer derselbe, unabhängig davon ob die TSE von fiskaltrust bezogen wird oder von einem anderen Reseller - wobei wir uns natürlich noch mehr darüber freuen, wenn auch die TSE von uns bezogen wird:
+
+- Erstellung einer SCU im Portal mit dem Package passend für den Hersteller der mitgebrachten TSE
+- Diese SCU muß mit der entsprechenden Cashbox verbunden werden
+- Die TSE kann in Verbindung mit unserer kostenfreien fiskaltrust.Middleware verwendet und mit Addons bzw. Produkt-Bundles, wie z.B. dem fiskaltrust.Sorglos ohne TSE Bundle, ergänzt werden.
```

Comment:
We should include the information about the credentials of the old TSE. These has to be known and entered in our configuration, since not the TSE gets blocked.